pkg/internal/controller/controller.go

```diff
@@ -228,11 +230,14 @@ func (c *Controller) reconcileHandler(obj interface{}) bool {
 		return true
 	}
 
+	log := c.Log.WithValues("name", req.Name, "namespace", req.Namespace)
+	ctx := logf.SetContext(context.Background(), log)
+
 	// RunInformersAndControllers the syncHandler, passing it the namespace/Name string of the
 	// resource to be synced.
-	if result, err := c.Do.Reconcile(req); err != nil {
+	if result, err := c.Do.Reconcile(ctx, req); err != nil {
 		c.Queue.AddRateLimited(req)
-		c.Log.Error(err, "Reconciler error", "name", req.Name, "namespace", req.Namespace)
+		log.Error(err, "Reconciler error")
```

Comment:
Thoughts about removing this? I'd prefer that we don't log for users if they don't want to, or if verbosity isn't high enough

Reply:
Even though i don't like this particular log, removing it looks like a breaking change.
Users probably rely on this log message & don't have a own logging of errors.

Reply:
+1 on increased verbosity from me by default, I think it would help in debugging